hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/CallIdSequenceWithBackoff.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
ok. I just pushed WIP doing exactly that - the same sequence is used by both client and member sides